MdeModulePkg/Bus/Pci/PciBusDxe/PciEnumeratorSupport.c

```diff
@@ -57,8 +57,13 @@ PciDevicePresent (
                                   1,
                                   Pci
                                   );
-
-  if (!EFI_ERROR (Status) && ((Pci->Hdr).VendorId != 0xffff)) {
+  // MU_CHANGE Begin
+  // The host bridge may be programmed to accept CRS.  If the PCI device is slow, and CRS is enabled,

```

Comment:
Since this has two approvals, I wanted to ask before being able to do a bit more investigation into the call stack.

Does it makse sense for a unique status code like `EFI_NOT_READY` to be returned? Right now `EFI_NOT_FOUND` could mean legitimately not found which is different than "not ready yet", right?

Reply:
I think that depends upon the requirement for the change.  If the desire is supporting CRS, then that is a bigger change and would require more control (ie what to do about NOT_READY - spin? For how long?)  The short term fix is to allow CRS to be enabled, and fail gracefully.  The current problem is that PciBus treats 0x0001 as a valid ID, and uses 0xFFFF from all the other config registers as the resources required.  So, this fix makes 0x0001 behave the same as 0xFFFF and return NOT_FOUND.

Reply:
Added debug warning message.